### PR TITLE
@alloy -> add conditions of sale button

### PR DESCRIPTION
**Pull request by 1aurabrown**
closes https://github.com/artsy/eigen/issues/357

Changes the name of the `how auctions work` link and added a link to the conditions of sale. I'm not sure if we want the conditions of sale to show up on works other than works at auction, but for now I made it uses the same logic as all the other auction controls.

Also tried to make the code that sets up all the buttons in the artwork view a little easier to read -- i was getting confused when looking at it, I think I made it a little easier.

### CODE REVIEW

File: Artsy/Classes/Views/ARArtworkActionsView.m [hunk 1]
```diff
@@ -142,19 +145,26 @@ - (void)updateUI
...
+
+    if ([self showConditionsOfSale]) {
+        ARInquireButton *auctionsInfo = [[ARInquireButton alloc] init];
+        [auctionsInfo setTitle:@"Conditions of Sale" forState:UIControlStateNormal];
+        [auctionsInfo addTarget:nil action:@selector(conditionsOfSaleTapped:) forControlEvents:UIControlEventTouchUpInside];
+
+        [self addSubview:auctionsInfo withTopMargin:isTopButton ? @"0" : @"8" sideMargin:nil];
     }
 
     NSArray *navigationButtons = [self navigationButtons];
```
Comment (alloy) [inline Artsy/Classes/Views/ARArtworkActionsView.m]: This whole section is getting very noisy and hard to follow. Do you see opportunities for refactoring and less repetition of hardcoded margin values as well? (so it only has to be replaced in 1 place)

Comment (1aurabrown) [inline Artsy/Classes/Views/ARArtworkActionsView.m]: Agreed. It was worse before. I could do this pretty easily if I ditched ORStackView and just used FLKAutoLayout.

Comment (orta) [inline Artsy/Classes/Views/ARArtworkActionsView.m]: It might be better to start splitting this class into more

Comment (alloy) [inline Artsy/Classes/Views/ARArtworkActionsView.m]: Oh interesting, what aspect is it that would make it easy if switching ORStackView->FLKAutoLayout? Is it the lack of ‘bottom margin’ in ORStackView?

Comment (1aurabrown) [inline Artsy/Classes/Views/ARArtworkActionsView.m]: Yes, but then again we'd have the same issue with figuring out if the given button is the last button, and then adjusting it's bottom margin accordingly.

I'm going to add a couple things to orstackview that may make things easier in this situation. Ideas include -- BOOL to force the first top margin to be zero or some number that you set on the stack view itself; method to update the top constraint of a given view after it's been added -- i.e. after adding all the buttons with their default top margins you can then go back and set the top margin of the first button (which may be any one of the buttons possibly added) to zero.

Right now I'm testing out that second idea.

Comment (1aurabrown) [inline Artsy/Classes/Views/ARArtworkActionsView.m]: i know orstackview has been updated, and this would diverge from version 2.0.0. I can make versions that work with strings and with floats if you think that makes sense, or i can just make it to use floats and try to update our app to the new orstackview api.

Comment (orta) [inline Artsy/Classes/Views/ARArtworkActionsView.m]: Think migrating ASAP is a good idea FWIW

**Comment by 1aurabrown** [comment]
can we merge this as-is, and I can follow up with some updates, most likely including a thing or two being added to orstackview?

**Comment by alloy** [comment]
@1aurabrown Sounds fair :+1:
